common/src/KokkosFFT_padding.hpp

```diff
@@ -117,8 +117,8 @@ void _crop_or_pad(const ExecutionSpace& exec_space, const InViewType& in,
 }
 
 template <typename ExecutionSpace, typename InViewType, typename OutViewType>
-void _crop_or_pad(const ExecutionSpace& exec_space, const InViewType& in,
-                  OutViewType& out, shape_type<2> s) {
+void _crop_or_pad(const ExecutionSpace&, const InViewType& in, OutViewType& out,
```

Comment:
```suggestion
void _crop_or_pad(const ExecutionSpace& exec_space, const InViewType& in, OutViewType& out,
```

Actually, an unused `exec_space` instance is a bug. Could you please rebase on PR #108.